### DISCUSSION TITLE
Too many Errors

### DISCUSSION THREAD

**Discussion by SunilKukreja1004** [akshayp7/playwright-typescript-playwright-test#38] [General]
Getting Errors in Playwright config TS regarding process as it is not defined cannot find module '@playwright/test';

**Comment by akshayp7** [comment]
Hi thanks for your feedback. Have you pulled the latest version.
Have you performed npm install after that?

**Comment by SunilKukreja1004** [comment]
Yes  I have done as mentioned
________________________________

**Comment by akshayp7** [comment]
@SunilKukreja1004  Can you please check now with latest verion 0.21.0 and let me know if issue still persists
